package.json

```diff
@@ -12,7 +12,9 @@
     "react-bootstrap": "^0.32.4",
     "react-dom": "^16.7.0",
     "react-redux": "^6.0.0",
-    "react-scripts": "2.1.3"
+    "react-scripts": "2.1.3",
+    "redux": "^4.0.1",
+    "redux-thunk": "^2.3.0"
```

Comment:
Never used this library, but from what I'm reading it helps with async requests. I've used `redux-promise-middle`, so hopping transitioning to this won't be that difficult. We'll all need to look into this.